File: Python/2850.py
while True:
    try:

        leg = input()

        if leg == "direita":
            print("frances")
        elif leg == "esquerda":
            print("ingles")
        elif leg == "nenhuma":
            print("portugues")
        else:
            print("caiu")

    except EOFError:
        break

File: C/2850.c
#include <stdio.h>
#include <string.h>

int main(){

    char leg[8];

    while( scanf("%[^\n]%*c", leg) != EOF){
        if(strcmp(leg, "direita") == 0){
            printf("frances\n");
        }else if(strcmp(leg, "esquerda") == 0){
            printf("ingles\n");
        }else if(strcmp(leg, "nenhuma") == 0){
            printf("portugues\n");
        }else{
            printf("caiu\n");
        }
    }
    
    return 0;
}